giraffe/src/components/TableCell.tsx

```diff
@@ -167,16 +167,16 @@ const getContents = (props: Props): string => {
   if (data && dataType.includes('dateTime')) {
     return timeFormatter(data)
   }
-
   if (
     isString(data) &&
     isFieldName(isVerticalTimeAxis, rowIndex, columnIndex)
   ) {
     return defaultTo(getFieldName(props), '').toString()
   }
 
-  if (!isNaN(+data) && decimalPlaces.isEnforced && decimalPlaces.digits < 100) {
-    return (+data).toFixed(decimalPlaces.digits)
+  if (!isNaN(+data)) {
+    // method needs the first arg to be a number to work properly
+    return formatStatValue(+data, {decimalPlaces})
```

Comment:
The whole function `getContents` probably needs unit tests to ensure that when `data` is `null` or empty string, the desired output is given.

Coercion is happening here to accommodate both `1234` and `"1234"` (as examples), but then coercion fails when we have an empty string.

Reply:
And to clarify: an empty string should show as an empty string.

Right now, and even with this code change, it will show as `0` which is not accurate. (This is the failure that I referred to above.)

Reply:
don't need to fix the !Nan coercion.  but yes, empty string should yield empty string from this function.  adding tests and fixing that problem *before* it gets to this clause.